src/len_zero.rs

```diff
@@ -30,21 +30,18 @@ impl LintPass for LenZero {
 impl LateLintPass for LenZero {
     fn check_item(&mut self, cx: &LateContext, item: &Item) {
         match item.node {
-            ItemTrait(_, _, _, ref trait_items) =>
-                check_trait_items(cx, item, trait_items),
-            ItemImpl(_, _, _, None, _, ref impl_items) => // only non-trait
```

Comment:
That comment is in a *very* strange place anyway.
(Should still exist, but should go down a bit)


Rustfmt needs to explicitly handle comments everywhere. Not ideal; but I don't know how to fix it.